src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -27,12 +27,23 @@
 namespace genny::v1 {
 namespace {
 
+class ParameterExpression {
+public:
+    static UniqueExpression parse(YAML::Node node, DefaultRandom& rng);
+};
+
+UniqueExpression ParameterExpression::parse(YAML::Node node, genny::DefaultRandom& rng) {
+    throw InvalidValueGeneratorSyntax(
+        "Encountered unexpected '^Parameter'. This indicates a bug in genny's parameter parsing "
```

Comment:
What is "genny's parameter parsing code"? Can we name a function here? Should the concept of `"ExternalPhaseConfig"` be mentioned here?

Reply:
Good point. This error doesn't necessarily mean a bug in genny.